phantom.coffee

```diff
@@ -21,10 +21,8 @@ startPhantomProcess = (binary, port, hostname, args) ->
 # @Description: kills off all phantom processes within spawned by this parent process when it is exits
 onSignal = ->
   phantom.exit() for phantom in phanta
-  process.exit()
```

Comment:
Because of this removed line, I can no longer kill a node process with Ctrl-C nor `kill`, it only kills the PhantomJS child processes, but the node process stays. I must do a `kill -9` to kill it, but obviously, the child processes are not killed.